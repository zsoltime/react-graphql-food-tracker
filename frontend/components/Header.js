import Link from 'next/link';
import React from 'react';

import { Logo, StyledHeader } from './Header.styles';
import Nav from './Nav';

const Header = () => (
  <StyledHeader>
    <nav className="navbar">
      <Logo>
        <Link href="/">
          <a>FoodTracker</a>
        </Link>
      </Logo>
      <Nav />
    </nav>
  </StyledHeader>
);

export default Header;
