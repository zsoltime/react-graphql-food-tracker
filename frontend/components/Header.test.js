import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Header from './Header';

describe('<Header />', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Header />);

    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
