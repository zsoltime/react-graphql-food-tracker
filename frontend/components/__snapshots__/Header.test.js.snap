// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> matches snapshot 1`] = `
<Headerstyles__StyledHeader>
  <nav
    className="navbar"
  >
    <Headerstyles__Logo>
      <Link
        href="/"
      >
        <a>
          FoodTracker
        </a>
      </Link>
    </Headerstyles__Logo>
    <Nav />
  </nav>
</Headerstyles__StyledHeader>
`;
